reference/cloud-access.md

```diff
@@ -0,0 +1,41 @@
+# Cloud project access
+
+We manage many projects across multiple cloud providers. This
+document defines our access policy, and is the canonical location
+for the projects we *do* have access to.
+
+## Access policy
+
+Every 2i2c engineer should have equal access to every cloud project
+we maintain. This prevents particular individuals from becoming
+single points of failure.
+
+In some cases, this requires paperwork for each engineer to get
+access. We should try to find ways around this, but if not,
+just do the paperwork.
+
+## Google Cloud Access
+
+On Google Cloud, we have a [2i2c organization](https://console.cloud.google.com/projectselector2/home/dashboard?organizationId=184174754493&supportedpurview=project),
+that contains some projects we are fully responsible for. Access
+to *all* projects in this organization can be granted by
+adding the user to the group `gcp-organization-admins@2i2c.org`
+in the [Google Workspace Admin dashboard](https://admin.google.com/ac/users).
```

Comment:
When I try to access that link, it shows me the Choose Account page but selecting the 2i2c.org email is not going forward, any thoughts?

Reply:
Only 2i2c.org workspace admins can access this - I've clarified that.

Reply:
> I've clarified that in the doc now.

Thanks!